README.md

```diff
@@ -17,7 +17,7 @@ A package ([documentation](https://biomarkersparkinson.github.io/tsdf/)) to load
 
 ### Using `pip`
 
-The package is available in PyPi. It can be installed using:
+The package is available in PyPi and requires [Python 3.9](https://www.python.org/downloads/) or higher. It can be installed using:
```

Comment:
Any specific reason for this requirement?

Reply:
https://github.com/biomarkersParkinson/tsdf/blob/8d44032b6f3290814d9bd1c5ccdb391256a8cd11/pyproject.toml#L19 also see #46